scanner/windows.go

```diff
@@ -5044,11 +5053,51 @@ func (w *windows) scanLibraries() (err error) {
 			// set dummy filemode because Windows file permission is complex and converting them to unix permission is difficult
 			filemode := os.FileMode(0666)
 
-			r := w.exec(w.translateCmd(fmt.Sprintf("Get-Content %s", abspath)), priv)
+			r := w.exec(w.translateCmd(func() string {
+				switch w.shell.psVersion {
+				case "":
+					return fmt.Sprintf("[System.IO.File]::ReadAllBytes(\"%s\")", abspath)
+				default:
+					v1, err := version.NewVersion(w.shell.psVersion)
+					if err != nil {
+						return fmt.Sprintf("[System.IO.File]::ReadAllBytes(\"%s\")", abspath)
+					}
+
+					v2, err := version.NewVersion("6.0")
+					if err != nil {
+						return fmt.Sprintf("[System.IO.File]::ReadAllBytes(\"%s\")", abspath)
+					}
+
+					if v1.LessThan(v2) {
+						return fmt.Sprintf("Get-Content -Path %s -Encoding Byte -Raw", abspath)
+					}
+					return fmt.Sprintf("Get-Content -Path %s -AsByteStream", abspath)
```

Comment:
```suggestion
						return fmt.Sprintf("Get-Content -Path \"%s\" -Encoding Byte -Raw", abspath)
					}
					return fmt.Sprintf("Get-Content -Path \"%s\" -AsByteStream", abspath)
```

Reply:
Looking at the documentation from Microsoft, it seems that quotes are not necessary.
https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.management/get-content?view=powershell-5.1#example-8-get-file-contents-as-a-byte-array
https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.management/get-content?view=powershell-7.5#example-8-get-file-contents-as-a-byte-array

Reply:
If the path contains spaces, it must be quoted.
```powershell
PS C:\Users\vagrant> Get-Content -Path test .txt
Get-Content : A positional parameter cannot be found that accepts argument '.txt'.
At line:1 char:1
+ Get-Content -Path test .txt
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidArgument: (:) [Get-Content], ParameterBindingException
    + FullyQualifiedErrorId : PositionalParameterNotFound,Microsoft.PowerShell.Commands.GetContentCommand
 
PS C:\Users\vagrant> Get-Content -Path 'test .txt'
PS C:\Users\vagrant>
```